designs/administrator/4.3/quay-image-security/index.md

```diff
@@ -0,0 +1,19 @@
+---
+parent: Administrator
+version: 4.3
+---
+
+# Quay Image Security Integration
+
+Users can now install the Container Security Operator to bring Quay and Clair metadata into OpenShift. Installing this operator enables cluster administrators to monitor known container image vulnerabilities in pods running on their Kubernetes cluster. After deployment, there is no additional configuration for the operator to start querying the container registry for vulnerability data. If the registry supports image scanning, like Quay, then the vulnerabilities found will be exposed to the user by populating the **ImageManifestVuln resource** list and surfacing it on the cluster dashboard.
+
+## Vulnerability details on the cluster dashboard
+
+![Cluster dashboard security card 1](img/image1-73.png)
+- Once the Container Security Operator has been installed, the **Quay Image Security** status is shown on the cluster dashboard Status card.
+- The card will list the number of vulnerabilities found and will provide a link to access more detailed information on the type of vulnerabilities.
+- Clicking on the link opens a popover with the breakdown of vulnerability by severity. Critical and high vulnerabilities will be listed first, followed by medium and lower risk. Fixable vulnerabilities are listed with two links.
+- One link is the vulnerability name that goes out to the vulnerability view for the Quay instance the image is hosted on (for example, Quay.io), and the other is a link to view the Image Manifest Vulnerability resource details in the affected namespace.
```

Comment:
I may be misunderstanding, but does the first link go to an external site? If so, we may want to consider showing an external link icon next to it!

Reply:
Great catch, seems like it should. I'm still undecided on whether post-implementation PRs should reflect reality or the ideal, but since this fix is so small, it may be worth updating the image.

Reply:
Agree great catch! Am working with Alec to get this added via bug. I will plan on updating the image.